### DISCUSSION TITLE
OneDrive 无api 挂载 webdav 会出现 PROPFIND /: 403 报错

### DISCUSSION THREAD

**Discussion by zwh310306** [AlistGo/alist#2089] [Q&A]
我用的OD，无api，企业版 。根据webdav的方法步骤来，结果出现 PROPFIND /: 403 报错。
用另一个有api的世纪互联版，使用od方法挂载就没事。 求解决

**Comment by Xcker8** [comment]
我用接口上传文件提示成功，但是没文件就很奇怪
